lib/ui/common/roundedButton.dart

```diff
@@ -53,28 +49,17 @@ class _DefaultStyle implements RoundedButtonStyle {
 }
 
 class RoundedButton {
```

Comment:
Remember... every widget should be (at least) a StatelessWidget if the widget can change its state then should be StatefulWidget.

Reply:
RoundedButton is modified as StatelessWidget in Isaac's branch [FARM-62-View-Prof-Loss-Statement]